### PR TITLE
chore: expose udh on short message

### LINKED ISSUE #33: How to concatenated sms?
https://en.wikipedia.org/wiki/Concatenated_SMS

### PR DESCRIPTION
**Pull request by septs**
closes #33

### CODE REVIEW

File: pdu/ShortMessage.go [hunk 1]
```diff
@@ -68,6 +68,12 @@ func (c *ShortMessage) SetLongMessageWithEnc(message string, enc data.Encoding)
 	return
 }
 
+// UDH get user data header for short message
+func (c *ShortMessage) UDH() UDH {
+	header := c.udHeader
+	return header
```
Comment (linxGnu) [inline pdu/ShortMessage.go:74]: `return c.udHeader`

**Review by linxGnu — CHANGES_REQUESTED** [review]
Thank you very much for contribution. Just a nits comment.

File: pdu/ShortMessage.go (hunk 1 above)
Comment (septs) [reply pdu/ShortMessage.go:74]: done

**Review by linxGnu — APPROVED** [review]

**Comment by linxGnu** [comment]
Thank you for PR @septs

**Pull request merged**